src/containers/Transactions/Description/AccountSet.jsx

```diff
@@ -10,14 +11,7 @@ const AccountSet = (props) => {
     lines.push(
       <div key="domain">
         {t('set_domain_description')}{' '}
-        <a
-          className="domain"
-          rel="noopener noreferrer"
-          target="_blank"
-          href={`http://${decodeHex(data.tx.Domain)}`}
-        >
-          {decodeHex(data.tx.Domain)}
-        </a>
+        <DomainLink className="domain" decode domain={data.tx.domain} />
```

Comment:
My guess is your test failure is on this line - `data.tx.domain` should be `data.tx.Domain` with a capital D :) 

(My thought process to get there was since the test failure said `decodeHex doesn't work on undefined` to look back through until I find something that could be undefined by accident)

Reply:
Yes, that seemed to fix the problem. Thanks haha.

Reply:
Meta: Once you've resolved all the comments from a reviewer, you can ask them to re-review in the top right corner of this page by clicking the "cycle" icon next to their username. 

<img width="312" alt="image" src="https://user-images.githubusercontent.com/19694186/187264487-61c69b35-a71d-44d2-b0b7-2eb523f804c8.png">